ui/views.py

```diff
@@ -154,6 +157,32 @@ def get_context_data(self, video, **kwargs):  # pylint: disable=arguments-differ
         return context
 
 
+class VideoDownload(VideoDetail):
+    """
+    Download a public video
+    """
+    def download(self, video):
```

Comment:
Can you add the `video` argument to the docstring?

Reply:
✅